lib/api-call.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const axios = require('axios');
+
+module.exports = {
+  getResponses: function(baseUrl, surveyID, endPoint, apiToken, apiTokenSecret) {
+    axios.get(baseUrl + '/' + surveyID + '/' + endPoint + '?api_token=' + apiToken + '&api_token_secret=' + apiTokenSecret)
+      .then(function (response) {
+        console.log(response);
+      })
+      .catch(function (error) {
+        console.log(error);
```

Comment:
Would be nice to have more sophisticated logging that just console.log but perhaps it probably doesn't matter for a first iteration.